app_dart/lib/src/service/scheduler.dart

```diff
@@ -382,6 +382,7 @@ class Scheduler {
       );
     } else {
       log.warning('Marking PR #$prNumber ci.yaml validation as failed');
+      log.warning(exception.toString());
```

Comment:
looks like we already set this as the `github.CheckRunOutput()`

Reply:
You can capture the backtrace on the catches up on lines 365 and 368, like:

```
} catch (e, b) {
  log.warning(e.toString());
  log.warning(b.toString());
  exception = e;
  backtrace = b;
}
```

Reply:
Thanks, done